### PR TITLE
add new badges to ReadMe

### PR DESCRIPTION
**Pull request by BenjaminW3**
Language, Platforms and License are nice looking new badges ;-)

### CODE REVIEW

**Comment by ax3l** [comment]
what about the osx platform? ;)

File: README.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 **alpaka** - Abstraction Library for Parallel Kernel Acceleration
 =================================================================
 
-[![Build Status](https://travis-ci.org/ComputationalRadiationPhysics/alpaka.svg?branch=develop)](https://travis-ci.org/ComputationalRadiationPhysics/alpaka)
-[![Build status](https://ci.appveyor.com/api/projects/status/xjeyugcg1cb0662s/branch/develop?svg=true)](https://ci.appveyor.com/project/BenjaminW3/alpaka-vuiya/branch/develop)
+[![Travis CI Build Status](https://travis-ci.org/ComputationalRadiationPhysics/alpaka.svg?branch=develop)](https://travis-ci.org/ComputationalRadiationPhysics/alpaka)
+[![Appveyor Build Status](https://ci.appveyor.com/api/projects/status/xjeyugcg1cb0662s/branch/develop?svg=true)](https://ci.appveyor.com/project/BenjaminW3/alpaka-vuiya/branch/develop)
```
Comment (ax3l) [inline README.md]: wrong repo selected

Comment (BenjaminW3) [inline README.md]: No, I do not think so. Appveyor seems to have the name of the user who created the appveyor account within its URL. This does not have any correspondence with the repos within my github account.

Comment (ax3l) [inline README.md]: huh, weird concept! thanks for the clarification.

**Pull request merged**